### ISSUE TITLE
Mimetype error with async functions

### ISSUE THREAD

**Issue by brett-cohen** [FireHead90544/craiyon.py#17]
The switch to CloudScraper worked for the standard `generate` function. However, the async functions are still throwing `Attempt to decode JSON with unexpected mimetype: text/html; charset=utf-8'`.

Recommend using a library like `aiocfscrape` to support CloudScraper's functionality for the async functions.

**Comment by mdm9300404** [comment]
I have been looking for something similar to `aiocfscrape` for such a long time. Thanks for the suggestion!

@FireHead90544 Think we should implement it?

**Comment by FireHead90544** [comment]
> The switch to CloudScraper worked for the standard `generate` function. However, the async functions are still throwing `Attempt to decode JSON with unexpected mimetype: text/html; charset=utf-8'`.
> 
> Recommend using a library like `aiocfscrape` to support CloudScraper's functionality for the async functions.

Oh so that's it, I totally forgot about the async-calls. That's a great idea.

**Comment by FireHead90544** [comment]
> I have been looking for something similar to `aiocfscrape` for such a long time. Thanks for the suggestion!
> 
> @FireHead90544 Think we should implement it?

Yea, I guess we should go with it.

**Comment by sionit1337** [comment]
How to solve this problem if my discord bot returns this error?

**Comment by FireHead90544** [comment]
> How to solve this problem if my discord bot returns this error?

Will be updating the logic for async calls soon using `aiocfscrape` or alternative probably

**Comment by sionit1337** [comment]
> Will be updating the logic for async calls soon using `aiocfscrape` or alternative probably

And when will the fix be released?

**Comment by truongquangtung2020** [comment]
Can you make an example for me pls?

**Comment by FireHead90544** [comment]
Refer to this comment regarding more details: https://github.com/FireHead90544/craiyon.py/issues/19#issuecomment-1906038250

TL;DR: Removed the cloudscraper dependency and switched to using headers. v3 works normally, v1 *sometimes* fails. 

Update the wrapper by running: `pip install -U craiyon.py`

Reopen the issue if this issue still persists
